lib/tmux_adapter.sh

```diff
@@ -1,12 +1,32 @@
 # Get the current path in the segment.
-get_tmux_cwd() {
-	local env_name=$(tmux display -p "TMUXPWD_#D" | tr -d %)
-	local env_val=$(tmux show-environment | grep --color=never "$env_name")
-	# The version below is still quite new for tmux. Uncomment this in the future :-)
-	#local env_val=$(tmux show-environment "$env_name" 2>&1)
-
-	if [[ ! $env_val =~ "unknown variable" ]]; then
-		local tmux_pwd=$(echo "$env_val" | sed 's/^.*=//')
-		echo "$tmux_pwd"
+MIN_MAJOR_VERSION="2"
+MIN_MINOR_VERSION="1"
+TMUX_VERSION="$(tmux -V)"
```

Comment:
I know `tmux -V` is quite not, I think it was not added until `v1.9` or so, but maybe the check at line 6 handles this anyways, simply by not matching the patterns?  :)